teuthology/provision.py

```diff
@@ -153,9 +161,19 @@ def build_config(self):
             'arch': 'x86_64',
         }
         fqdn = self.name.split('@')[1]
-        file_out = {'downburst': file_info, 'local-hostname': fqdn}
+        file_out = {
+            'downburst': file_info,
+            'local-hostname': fqdn,
+        }
         yaml.safe_dump(file_out, config_fd)
         self.config_path = config_fd.name
+
+        user_info = {
+            'user': 'ubuntu',
```

Comment:
Instead of hardcoding this, could we add an arg to ``__init__`` and default it to 'ubuntu'?

Also, has this been tested with existing downburst images?